docs/CreateEdit.md

```diff
@@ -946,6 +949,7 @@ const validateEmail = email();
 const validateAge = [number(), minValue(18)];
 const validateZipCode = regex(/^\d{5}$/, 'Must be a valid Zip Code');
 const validateSex = choices(['m', 'f'], 'Must be Male or Female');
+const validateDate = [minDate(new Date())];
```

Comment:
Should use maxDate here to make this exemple more consistant with a Birthdate

Reply:
you should also add formatting options to illustrate usage